### ISSUE TITLE
Printing and transparency

### ISSUE THREAD

**Issue by Albert-Bujanovs** [aardappel/treesheets#80]
I'm having issues with printing both to paper and PDF. It seems the PNG alpha channel is not rendered correctly. For instance the tutorial document has the nested style for tables but when printed the corner lines have black boxes. The same happens with every other icon available in the program. I'm using Windows 10 with the latest updates. Additionally when saving HTML with styles it doesn't save the images, unfortunately making TreeSheets unusable for a production environment. Can anyone else confirm this issue? Thanks.

**Comment by aardappel** [comment]
Yes, the printing code in wxWidgets has always been broken compared to bitmap rendering. The only way to fix that (apart from trying to fix wxWidgets) is to have separate rendering paths for when using screen vs print, and fix issues 1 by 1. This can be done, but is a fair bit of work.

A workaround is to export your view to an image, then print the image. Lower quality, but at least will no rendering issues.
